apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -1,5 +1,7 @@
 import { Trans } from "next-i18next";
 import Link from "next/link";
+// eslint-disable-next-line @calcom/eslint/deprecated-imports-next-router
```

Comment:
For reviewers: we are using next.js at version 13 and "next/router" has been deprecated, so we should import "useRouter" from "next/navigation", but "router.events" does not exist in version 13 so looks like need to roll with the deprecated import.

https://github.com/vercel/next.js/discussions/42016?sort=new